### PR TITLE
Python Version of "MAVROS Offboard Control Example" with more detailed instructions for beginners

### PR DESCRIPTION
**Pull request by joao-pinhal-feup**
Hello, so as I discussed one day in the PX4 slack with some contributors, being a complete beginner to the PX4 Firmware I found the documentation extremely confusing and some parts terribly over complicated or with a lot of steps missing. As someone who just wanted to test a simple program in Python with ROS who would just follow some waypoints, I spent probably 2 months just trying to install it and figuring it out. This contribution doesn't fix the problems with installation, but I hope it helps a lot of beginners like I was.

Basically just added some much needed step-by-step instructions for the Offboard example and since I was more familiar with Python I "translated" it from the C++ of the original page.

I hope to contribute more, specially for the beginner folks, but since I am currently doing my master's thesis this is the most useful thing I can do right now.

### CODE REVIEW

File: en/flight_modes/offboard.md [hunk 1]
```diff
@@ -142,17 +142,18 @@ The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL
...
 
 Typically developers do not directly work at the MAVLink layer, but instead use a robotics API like [MAVSDK](https://mavsdk.mavlink.io/) or [ROS](http://www.ros.org/) (these provide a developer friendly API, and take care of managing and maintaining connections, sending messages and monitoring responses - the minutiae of working with *Offboard mode* and MAVLink). 
 
 The following resources may be useful for a developer audience:
 
-* [Offboard Control from Linux](../ros/offboard_control.md) (PX4 Devguide)
-* [MAVROS Offboard control example](../ros/mavros_offboard.md) (PX4 Devguide)
+* [Offboard Control from Linux](../ros/offboard_control.md)
+* [ROS/MAVROS Offboard Example (C++)](ros/mavros_offboard_cpp.md)
+* [ROS/MAVROS Offboard Example (Python)](ros/mavros_offboard_python.md)
```
Comment (hamishwillee) [inline en/flight_modes/offboard.md:159]:
```suggestion
* [ROS/MAVROS Offboard Example (C++)](../ros/mavros_offboard_cpp.md)
* [ROS/MAVROS Offboard Example (Python)](../ros/mavros_offboard_python.md)

```

File: en/ros/mavros_offboard_python.md [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+
+rospy.wait_for_service("/mavros/cmd/arming")
+arming_client = rospy.ServiceProxy("mavros/cmd/arming", CommandBool)    
+
+rospy.wait_for_service("/mavros/set_mode")
+set_mode_client = rospy.ServiceProxy("mavros/set_mode", SetMode)
+```
+
+PX4 has a timeout of 500ms between two *OFFBOARD* commands.
+If this timeout is exceeded, the commander will fall back to the last mode the vehicle was in before entering *OFFBOARD* mode.
```
Comment (hamishwillee) [inline en/ros/mavros_offboard_python.md:192]: Have you tested this? I only ask because I thought offboard fell back to Hold mode.

**Comment by hamishwillee** [comment]
Hi @joao-pinhal-feup 

Thanks so much - t his looks excellent. I have done a minor subedit to fix up a few links and update some "layout to my taste".

Merging!

Cheers
Hamish

PS Just FYI for next time, when working with git, it is best to fork the repo (as you have done) then make your modifications in a branch rather than main. If you do that, it makes it easier to manage multiple contributions (i.e. because this was called "main" I had to do the edits in github rather than locally).

**Comment by joao-pinhal-feup** [comment]
Sorry for the late reply and that inconvenience, it is my first time contributing to open source so I did not know about that.

File: en/ros/mavros_offboard_python.md (hunk 2 above)
Comment (joao-pinhal-feup) [reply en/ros/mavros_offboard_python.md:192]: I did test it and the same was written in the  C++ example.

**Comment by hamishwillee** [comment]
No worries. It's mostly a "git" thing. Atlassian have some good docs on this here (ignore that it is for bitbucket, the behaviour is the same for github) https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow#:~:text=Forking%20is%20a%20git%20clone,org%2FuserA%2Fopen%2Dproject
